en/xml/fo/topic.graphic-pageheader.xml

```diff
@@ -0,0 +1,232 @@
+                                coords="19"/>
+                            <area xml:id="co.topic.graphic-pageheader.content"
+                                coords="32"/>
+                            <area xml:id="co.topic.graphic-pageheader.block1"
+                                coords="39"/>
+                            <area xml:id="co.topic.graphic-pageheader.center"
+                                coords="42"/>
+                             <area xml:id="co.topic.graphic-pageheader.right"
+                                coords="45"/>
+                            <area xml:id="co.topic.graphic-pageheader.height"
+                                coords="47 56" units="linecolumn"/>
+                            <area xml:id="co.topic.graphic-pageheader.attribute"
+                                coords="53"/>
+                            <area xml:id="co.topic.graphic-pageheader.template"
+                                coords="50 56" units="linecolumn"/>
+                            <area xml:id="co.topic.graphic-pageheader.file"
+                                coords="53" units="linecolumn"/>
+                        </areaspec>
+                        <programlisting language="xml"
+                            linenumbering="numbered"><xi:include
+                                href="graphic-pageheader/graphic-pageheader_shortened.xsl"
+                                parse="text"/></programlisting>
+                    </programlistingco>
+                </example>
+                <calloutlist>
+                    <callout arearefs="co.topic.graphic-pageheader.path">
+                        <para>The directory path to your logos.</para>
+                    </callout>
+                    <callout arearefs="co.topic.graphic-pageheader.variables">
+                        <para>This is the default value and will be used if no
+                            parameters are passed.</para>
+                    </callout>
+                    <callout arearefs="co.topic.graphic-pageheader.content">
+                        <para>We assume that your logo should be placed in the
+                            header of all pages except title pages. You can
+                            switch single- or double-sided with this variable.</para>
+                    </callout>
+                    <callout arearefs="co.topic.graphic-pageheader.block1">
+                        <para>A <tag>fo:block</tag>-element creates a rectangle
+                            area on a page which can contain lists, tables or
+                            graphics. In this demo, the document title is
+                            visible on the left side of single-sided documents.</para>
+                    </callout>
+                    <callout arearefs="co.topic.graphic-pageheader.center">
+                        <para>In this demo, the center position has no content,
+                            but you can add a page number for example.</para>
```

Comment:
I would rephase the above paragraph slightly to mention the element `<fo:page-number/>`:

```xml
... but you can add the page number with the element
<tag class="emptytag">fo:page-number</tag>, for example.
```